infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -315,43 +315,50 @@ function ensure_all_prod_special_cases() {
 
     # Special case: create/add-permissions for necessary service accounts for the auditor.
     color 6 "Empowering artifact auditor"
-    empower_artifact_auditor "${PROD_PROJECT}"
-    empower_artifact_auditor_invoker "${PROD_PROJECT}"
+    empower_image_auditor "${PROD_PROJECT}"
+    empower_image_auditor_invoker "${PROD_PROJECT}"
 
     # Special case: give Cloud Run Admin privileges to the group that will
     # administer the cip-auditor (so that they can deploy the auditor to Cloud Run).
     color 6 "Empowering artifact-admins to release prod auditor"
-    empower_group_to_admin_artifact_auditor \
+    empower_group_to_admin_image_auditor \
         "${PROD_PROJECT}" \
         "k8s-infra-artifact-admins@kubernetes.io"
 
     # TODO: what is this used for?
     color 6 "Ensuring prod promoter vuln scanning svcacct exists"
     ensure_service_account \
         "${PROD_PROJECT}" \
-        "${PROMOTER_VULN_SCANNING_SVCACCT}" \
+        "${IMAGE_PROMOTER_VULN_SCANNING_SVCACCT}" \
         "k8s-infra container image vuln scanning"
 
     # Special case: allow prow trusted build clusters to run jobs as
     # prod-related GCP service accounts
     color 6 "Empowering trusted prow build clusters to use prod-related GCP service accounts"
     for project in "${PROW_TRUSTED_BUILD_CLUSTER_PROJECTS[@]}"; do
+        # Grant write access to k8s-artifacts-prod GCS
+        serviceaccount="$(svc_acct_email "${PROD_PROJECT}" "${FILE_PROMOTER_SVCACCT}")"
+        color 6 "Ensuring GKE clusters in '${project}' can run pods in '${PROWJOB_POD_NAMESPACE}' as '${serviceaccount}'"
+        empower_gke_for_serviceaccount \
+            "${project}" "${PROWJOB_POD_NAMESPACE}" \
+            "${serviceaccount}" "k8s-infra-promoter"
```

Comment:
yeah actually this will be unhappy since the service account doesn't exist unless something calls `empower_file_promoter`

Reply:
Fixed.
Now being called as part of `ensure_prod_gcs_bucket()`.